### PR TITLE
Add minutes for 2024-12-03

### PR DESCRIPTION
**Pull request by 40thieves**
## What does this change?

Adds Syllabus Team minutes for 2024-12-03

## Who needs to know about this?

@CodeYourFuture/global-syllabus

### CODE REVIEW

File: org-cyf-guides/content/contributing/minutes/index.md [hunk 1]
```diff
@@ -17,6 +17,97 @@ Attendees:
...
 ---
 
+## 2024-12-03
+
+Attendees: Ali Smith, Sally McGrath, Mitch Lloyd
+
+### ❗Actions from last time
+
+* [ ] All: Collaboratively put together LOs and name for ITP Introduction Module
+* [ ] Daniel: Sync with Karen on Piscine dashboard requirements
```
Comment (illicitonion) [inline org-cyf-guides/content/contributing/minutes/index.md:27]:
```suggestion
* [x] Daniel: Sync with Karen on Piscine dashboard requirements
```

We did this! tl;dr: Short-term we're going to run the application process outside of the dashboard, and will use the dashboard to track progress _within_ the Piscine. Long-term they know what we need for application stuff too.

File: org-cyf-guides/content/contributing/minutes/index.md [hunk 2]
```diff
@@ -17,6 +17,97 @@ Attendees:
...
+  * Do _not_ want to explain why lecturing is a bad thing!
+    * Ali: Perhaps need to think about how we can filter out this sort of thinking before we get to this stage
+* Ali: Have been thinking about conference talks with the idea that we can try to attract people with a matching mindset & skills
+  * Have put in for several CFPs, but not got anything back yet
+* Sally: German to ask Ali about what CYF can do to help in this area
+  * Ali: The main thing is time!
+
+### ❗Actions
+
+* [ ] Daniel: Sync with Karen on Piscine dashboard requirements
```
Comment (illicitonion) [inline org-cyf-guides/content/contributing/minutes/index.md:105]:
```suggestion
```

File: org-cyf-guides/content/contributing/minutes/index.md [hunk 3]
```diff
@@ -17,6 +17,97 @@ Attendees:
...
+* Sally: Have started on a SAP FI/CO Track
+  * [Curriculum link](https://tracks.codeyourfuture.io/sap-fico/)
+  * If we land a deal with a large enterprise company, then we'll run this Track
+  * Financial / Enterprise Resource planning
+  * Will run as workshops on Saturdays, with volunteers from the company
+    * They'll also provide environments for trainees to use
+  * Anyone who completes, should be eligible for submitting their CV to the company
+* Sally: Working with employers to agree LOs, pre-reqs for getting the job
+  * [Example doc](https://docs.google.com/document/d/1zJomRiMshLluuWq_F_dM8fAxSM-ljmZZrl4H1o9aZxc/edit?tab=t.0)
+* Sally: Google have awarded us a grant to run an AI Essentials track
```
Comment (SallyMcGrath) [inline org-cyf-guides/content/contributing/minutes/index.md:69]:
```suggestion
* Sally: Centre for Public Impact have awarded us a grant to run an AI Essentials track
```

File: org-cyf-guides/content/contributing/minutes/index.md [hunk 4]
```diff
@@ -17,6 +17,97 @@ Attendees:
...
+  * [Press release](https://centreforpublicimpact.org/ai-opportunity-fund/)
+  * Will run by Google, essentially it's a version of their online video course
```
Comment (SallyMcGrath) [inline org-cyf-guides/content/contributing/minutes/index.md:71]:
```suggestion
  * Will run by their provided trainers, essentially it's a version of their online video course
```

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
Sorry just a contractual agreement change based on how we describe the grant. https://centreforpublicimpact.org/ai-opportunity-fund/

File: org-cyf-guides/content/contributing/minutes/index.md (hunk 1 above)
Comment (40thieves) [reply org-cyf-guides/content/contributing/minutes/index.md:27]: Thanks! I had a feeling you had, I remember seeing slack activity about it, but didn't want to complete without double checking.

**Review by illicitonion — APPROVED** [review]

**Pull request merged**